diff --git a/codex-rs/rmcp-client/Cargo.toml b/codex-rs/rmcp-client/Cargo.toml
index 517f727..839e027 100644
--- a/codex-rs/rmcp-client/Cargo.toml
+++ b/codex-rs/rmcp-client/Cargo.toml
@@ -32,7 +32,6 @@ oauth2 = "5"
 reqwest = { version = "0.13", default-features = false, features = [
     "json",
     "stream",
-    "rustls",
 ] }
 rmcp = { workspace = true, default-features = false, features = [
     "auth",
